src/axom/sidre/core/Group.hpp

```diff
@@ -154,6 +147,28 @@ class Group
    */
   char getPathDelimiter() const { return s_path_delimiter; }
 
+  /*!
+   * \brief static method to get valid protocols for Group I/O methods.
+   *
+   * Only protocols that work for both input and output are returned.
+   */
+  static const std::vector<std::string>& getValidIOProtocols()
+  {
+    return s_io_protocols;
+  }
+
+  /*!
+   * \brief static method to get the default I/O protocol.
+   */
+  static std::string getDefaultIOProtocol()
```

Comment:
Is there a reason not to follow the same pattern for this method as the one above; i.e., define a class member based on build option and return a cons string ref here?

Reply:
That was how I originally did it, but then I changed it to the current thing in response to a suggestion to define it here rather than carry around a static member.